src/Calculator.cpp

```diff
@@ -7,5 +7,23 @@
 ////////////////////////////////////////////////////////////////////////////
 
 #include "Calculator.h"
+// Add Function definitions here.
 
-// Add Function definitions here.
\ No newline at end of file
+void printCalculations(int a, int b)
+{
+    std::cout << std::endl;
+    std::cout << formatCalculation(a, b, a + b, '+') << std::endl;
+    std::cout << formatCalculation(a, b, a * b, 'X') << std::endl;
+    std::cout << formatCalculation(a, b, b == 0 ? 0 : a / b, '%') << std::endl;
+    std::cout << formatCalculation(a, b, a - b, '-') << std::endl;
+}
+
+std::string formatCalculation(int a, int b, int result, char operation)
+{
+    //Sanity check if a control character gets inserted
+    if (operation < 32 || operation > 126)
```

Comment:
32 and 126 seem like magic numbers. If they are control character can declare them as constexpr or separate class and use them here